src/SFA.DAS.EmploymentCheck.Functions/Application/Services/EmployerAccount/EmployerAccountService.cs

```diff
@@ -56,135 +60,87 @@ public class EmployerAccountService
             _hostingEnvironment = hostingEnvironment;
             _azureClientCredentialHelper = azureClientCredentialHelper;
             _azureServiceTokenProvider = azureServiceTokenProvider;
+            _repository = repository;
         }
         #endregion Constructors
 
-        #region GetPayeSchemes
-
-        public async Task<ResourceList> GetPayeSchemes(Models.EmploymentCheck apprenticeEmploymentCheck)
+        public async Task<EmployerPayeSchemes> GetEmployerPayeSchemes(Models.EmploymentCheck employmentChecksBatch)
         {
-            var resourceList = await Get<ResourceList>(apprenticeEmploymentCheck);
-
-            return resourceList;
-        }
+            Guard.Against.Null(employmentChecksBatch, nameof(employmentChecksBatch));
 
-        #endregion GetPayeSchemes
+            var resourceList = await Get<ResourceList>(employmentChecksBatch);
+            if (!resourceList.GetType().Equals(typeof(ResourceList)))
```

Comment:
this gives out a warning `Check for reference equality instead`  recommending. `if (resourceList.GetType() != typeof(ResourceList))`

Reply:
Changed to use recommendation